docs/crd-changelog.md

```diff
@@ -0,0 +1,160 @@
+# CRD Changelog
+
+This document explains major changes made in new CRD versions. It is intended to help users migrate and take
+advantage of the new features.
+
+## OpenTelemetryCollector.opentelemetry.io/v1beta1 
+
+### Migration
+
+There is no need for any immediate user action. The operator will automatically convert existing `v1alpha1` resources to `v1beta1`.
```

Comment:
>  The operator will automatically convert existing `v1alpha1` resources to `v1beta1`.

I would rephrase this. Yes there is conversion webhook but for migration from v1alpha1 to v1beta1 an update on the CR must happen